packages/diagrams/frontend/sirius-components-diagrams-reactflow/src/renderer/edge/EdgeLabeL.types.ts

```diff
@@ -0,0 +1,19 @@
+/*******************************************************************************
+ * Copyright (c) 2023 Obeo.
+ * This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v2.0
+ * which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     Obeo - initial API and implementation
+ *******************************************************************************/
+import { GQLLabelStyle } from '../../graphql/subscription/labelFragment.types';
+
+export interface EdgeLabelProps {
+  transform: string;
+  label: string;
+  edgeLabelStyle: GQLLabelStyle;
```

Comment:
The props should contain the label just like for the node (and the transform property since that's required)